portfolio/src/main/webapp/leave-comment/leave-comment.html

```diff
@@ -12,18 +12,19 @@
   <body>
     <div id="leave-comment-form">
       Please share your feedback!
-      <form>
-        <input type="email" placeholder="Your email adress">
+      <form method="post" action="/add-comment">
+        <input name="email" type="email" placeholder="Your email adress">
         <textarea name="text" wrap="soft" placeholder="Your comment"></textarea>
         <input class="button" type="submit" placeholder="Submit!">
       </form>
     </div>
     <div id="sidebar">
       <div id="comment-list">
-        Here the comments will be displayed.
+        Comments:
       </div>
       <a class="button" href="../index.html">Main page</a>
```

Comment:
When one goes back to the main page via this button, the URL ends up being different (i.e. it's https://kluczek-step-2020.appspot.com/index.html instead of the initial https://kluczek-step-2020.appspot.com/).

Reply:
Done.